src/pages/Scheme/components/HierarchyLevelList/components/ContentEditable/ContentEditable.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useRef } from 'react';
+import ContentEditable from 'react-contenteditable';
+
+interface ContentEditableFieldProps {
+  onChange: (value: string) => void;
+  disabled?: boolean;
+  tag?: string;
+  className?: string;
+  value?: string;
+}
+
+const ContentEditableField: React.FC<ContentEditableFieldProps> = ({
+  value = '',
+  disabled,
+  tag = 'span',
+  className,
+  onChange,
+}) => {
+  const editableRef = useRef(null);
+  return (
+    <ContentEditable
+      innerRef={editableRef}
+      tagName={tag}
+      className={className}
+      disabled={disabled}
+      html={value}
+      onChange={(e) => {
+        const { value: str } = e.target;
+        onChange(str);
+      }}
```

Comment:
Мне кажется здесь можно сократить, только нужно уснать какой тип прилетает для эвента

```suggestion
      onChange={({ target }) => {
        onChange(target.value);
      }}
```

Reply:
done